src/code/z_parameter.c

```diff
@@ -4012,7 +4012,7 @@ void Interface_Update(PlayState* play) {
     s16 risingAlpha;
     u16 action;
 
-#if OOT_DEBUG
+#if OOT_PAL && DEBUG_FEATURES
```

Comment:
should we make an NTSC version for this too? Ditto with FileSelect_PulsateCursor. Should be an easy copy-paste job, although we'd have to guess which d-pad buttons to use (say d-left for jp and d-right for eng)

Reply:
Im not super keen on trying to make up code for what we *think* they may have had. I would probably prefer to just leave it, but down to hear others thoughts as well

Reply:
i'm strongly against this, as I believe decomp is not the appropriate place for custom mods, speedhacks, and the like. It should simply be representative of the base builds, free of custom modifications.

If you desperately need such a feature, you are always free to create your own personal fork.

Reply:
Yeah I would not want to add such code unless one day we do find an NTSC debug build that lets us objectively say what existed